code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1153,6 +1153,7 @@
 		..()
 		reagents.add_reagent("nutriment", 6)
 		reagents.add_reagent("bustanut", 6)
+		reagents.add_reagent("sodiumchloride", 6)

```

Comment:
I wanted to provide greytide with more ready access to salt if the chef is being uncooperative.

This amount of salt should be harmless. I thought Busta's salty nature made it fitting.